plugin/powerControl/predictor/predictor.py

```
@@ -0,0 +1,447 @@
+import os
+import yaml
+import logging
+import holidays
+import traceback
+import argparse
+import math
+import datetime
```

Ruff (0.8.2)

`datetime` imported but unused

Remove unused import: `datetime`

(F401)